public/javascripts/tests.js

```diff
@@ -84,17 +100,109 @@ function renderTestsList(jobs, is_operator, restart_url) {
 	    },
 	],
     } );
+
+    var scheduled_table = $('#scheduled').DataTable( {
+        "pagingType" : 'simple',
+        "order": [[3, 'asc'], [0, 'asc']],
+        "ajax": {
+            "url": "/tests/list_ajax",
+            "type": "POST", // we use POST as the URLs can get long
+            "data": function(d) {
+            var ret = {
+                "relevant": $('#relevantfilter').prop('checked'),
+                "state" : 'scheduled'
+            };
+            if (jobs != null) {
+                ret['jobs'] = jobs;
```

Comment:
Hmm.. as I look into it, I think I added 'state' param. It works in tests, but maybe not as I imagined. Will need to check.